docs/usage/dex.md

```diff
@@ -65,30 +67,41 @@ metadata:
   labels:
     example: openshift-oauth
 spec:
-  dex:
-    config: |
-      connectors:
-        # GitHub example
-        - type: github
-          id: github
-          name: GitHub
-          config:
-            clientID: xxxxxxxxxxxxxx
-            clientSecret: $dex.github.clientSecret # Alternatively $<some_K8S_secret>:dex.github.clientSecret
-            orgs:
-            - name: dummy-org
+  sso:
+    provider: dex
+    dex:
+      config: |
+        connectors:
+          # GitHub example
+          - type: github
+            id: github
+            name: GitHub
+            config:
+              clientID: xxxxxxxxxxxxxx
+              clientSecret: $dex.github.clientSecret # Alternatively $<some_K8S_secret>:dex.github.clientSecret
+              orgs:
+              - name: dummy-org
 ```
 
-## Disable DEX
+## Install/Uninstall DEX
 
-Dex is installed by default for all the Argo CD instances created by the operator. You can disable this behavior using the environmental variable `DISABLE_DEX` on the operator.
+Dex is not installed by default for any of the Argo CD instances created by the operator. It can be enabled by setting `.spec.sso.provider` to `dex` and supplying a non-empty `.spec.sso.dex` section within the Argo CD custom resource. For example: 
```

Comment:
We were deploying DEX OOTB for the Argo CD instances that are created from the Operator Console.
https://github.com/argoproj-labs/argocd-operator/blob/6de5fc2585acdcb595c04cf400139fac48eda431/bundle/manifests/argocd-operator.clusterserviceversion.yaml#L50


I dont think it is a good idea to change the behavior. What do you think ?